src/care/ProfilePlugin.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _CARE_ProfilePlugin_H_
+#define _CARE_ProfilePlugin_H_
+
+#include "RAJA/util/PluginStrategy.hpp"
+#include "chai/ExecutionSpaces.hpp"
+#include "care/config.h"
+#include "care/CHAICallback.h"
+#include "care/RAJAPlugin.h"
+
+using namespace care;
+
+namespace care{
+
+class RAJAPlugin;
+
+}
+
+namespace chai{
+
+	class ProfilePlugin : public RAJA::util::PluginStrategy
+	{
+		public:
+			ProfilePlugin();
+
+			char * fileName;
```

Comment:
Where are the fileName and lineNumber set?

Reply:
This is still not being set anywhere.

Reply:
i think those variables have since been deleted

Reply:
We still need the file and line number. Take a look at what you commented out in forall.h and think about how you can get the file and line number to this class.